### DISCUSSION TITLE
typer documentation as markdown

### DISCUSSION THREAD

**Discussion by petre-renware** [fastapi/typer#804] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# thanks for answer. It works now.
```

### Description

1. Thanks for answer and app update. It works now.

2. One more suggestion (not critical anymore, but "nice to be"): *when load and run code to create help and documentation*, **it wil be useful to redirect all `stdout` & `stderr` because the run application simply just can have output messages written unconditionally, messages that will appear in generated doc**. Thank you. A very nice and beautiful day to have, all of you !

### Operating System

Linux, Windows

### Typer Version

last @2024.apr.24

### Python Version

3.10

**Comment by YuriiMotov** [comment]
It's not clear from the context provided, but what you are probably talking about is described here: [Breaking completion in a callback](https://typer.tiangolo.com/tutorial/options/callback-and-context/#breaking-completion-in-a-callback)

If it's not what you were asking, please clarify your request.
I'll close this for now, but feel free to re-open if it's still relevant
